### DISCUSSION TITLE
Problem to read parameter

### DISCUSSION THREAD

**Discussion by jhoerter70** [Art-of-WiFi/UniFi-API-client#91] [General]
Hi,
i have a problem to read system-stats with php.
May be it is a problem of line in system-stats, normally the names has underline?

Code:
 echo $ap->system-stats=>cpu . PHP_EOL;
Error: 
 PHP Parse error:  syntax error, unexpected '=>' (T_DOUBLE_ARROW), expecting ';' or ',' in ....

"system-stats": {
            "cpu": "10.9",
            "mem": "40.9",
            "uptime": "28187"
        },

**Comment by malle-pietje** [comment]
I believe you need to use a single arrow instead of a double arrow like so:
```php
echo $ap->system-stats->cpu . PHP_EOL;
```

**Comment by jhoerter70** [comment]
I'm sorry the double arrow is from another test.

I correct it to 

echo $ap->system-stats->cpu . PHP_EOL; 

and now the error is: 

PHP Parse error:  syntax error, unexpected '->' (T_OBJECT_OPERATOR), expecting ';' or ',' in

**Comment by malle-pietje** [comment]
Ah OK. Then the issue is caused by the property name which contains a dash. Change the line to this:
```php
echo $ap->{"system-stats"}->cpu . PHP_EOL;
```

**Comment by jhoerter70** [comment]
That's was it. Thank you.
